src/engraving/libmscore/note.cpp

```diff
@@ -2089,6 +2085,17 @@ Element* Note::drop(EditData& data)
     break;
 
     default:
+        Spanner* spanner;
+        if (e->isSpanner() && (spanner = toSpanner(e))->anchor() == Spanner::Anchor::NOTE) {
+            spanner->setParent(this);
```

Comment:
couldn't find a nice way to avoid this one special case - note-anchored lines need to know exactly which note they're being pasted on to, and computeEndElement() needs to be called after setTrack2().
However it could be simplified if setTrack2( ) just triggered the call to computeEndElement() automatically...